docs/_docs/features/debugger.md

```diff
@@ -6,7 +6,7 @@ permalink: /docs/features/debugger/
 ---
 
 One of the key features of Nuclide is its multiple-language debugging support. The Nuclide Debugger
-is provided within the familiar [Chrome DevTools](https://developer.chrome.com/devtools) interface.
+is provided with the familiar [Chrome DevTools](https://developer.chrome.com/devtools) interface.
```

Comment:
I changed  this from within to with because there is a button to click that pulls up the Chrome Dev Tools..... but I could see where just removing this sentence would be find since it's mentioned later on in the doc.